.travis.yml

```diff
@@ -86,16 +54,8 @@ install:
       cabal new-update
```

Comment:
I wonder if exporting `$PATH` variable is still required with new settings for cabal and ghc? :thinking: